includes/wc-attribute-functions.php

```diff
@@ -321,3 +321,27 @@ function wc_attributes_array_filter_visible( $attribute ) {
 function wc_attributes_array_filter_variation( $attribute ) {
 	return $attribute && is_a( $attribute, 'WC_Product_Attribute' ) && $attribute->get_variation();
 }
+
+/**
+ * Check if an attribute is included in the attributes area of a product name.
+ *
+ * @since  3.0.2
+ * @param  string $attribute Attribute value to check for
+ * @param  string $name      Product name to check in
+ * @return bool
+ */
+function wc_is_attribute_in_product_name( $attribute, $name ) {
```

Comment:
This could be made more general purpose; `wc_product_name_contains( $name, $string)`   or similar?